buildSrc/src/main/groovy/org/robolectric/gradle/SpotlessPlugin.groovy

```diff
@@ -0,0 +1,40 @@
+package org.robolectric.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class SpotlessPlugin implements Plugin<Project> {
+    void apply(Project project) {
+        project.getPlugins().apply('com.diffplug.spotless')
+
+        project.spotless {
+            kotlin {
+                // Add configurations for Kotlin files
+                target '**/*.kt'
+                ktfmt('0.42').googleStyle()
+            }
+            java {
+                // Add configurations for Java files
+                if (project.name == 'buildSrc') {
+                    target("**/*.java")
+                } else {
+                    target("buildSrc/**/*.java")
+                }
```

Comment:
Isn't it enough to check for `**/*.java`? If so, same for Groovy?
```suggestion
                target("**/*.java")
```

Reply:
> Isn't it enough to check for `**/*.java`? If so, same for Groovy?

Thank you for reviewing....made suggested changes ..is there anything else that needs to be fixed?